build_tools/waf_dynamo.py

```diff
@@ -367,7 +367,7 @@ def default_flags(self):
     if 'osx' == build_util.get_target_os() or 'ios' == build_util.get_target_os():
         self.env.append_value('LINKFLAGS', ['-weak_framework', 'Foundation'])
         if 'ios' == build_util.get_target_os():
-            self.env.append_value('LINKFLAGS', ['-framework', 'UIKit', '-framework', 'SystemConfiguration', '-framework', 'AVFAudio'])
+            self.env.append_value('LINKFLAGS', ['-framework', 'UIKit', '-framework', 'SystemConfiguration', '-framework', 'AVFoundation'])
```

Comment:
Audio functionality split into `AVFAudio` from `AVFoundation` in iOS 14.5
But we should use AVFoundation to keep back compatibility. I'm not sure how it resolved inside SDK, pbbly `AVFAudio`  is linked in  `AVFoundation` or something like that